resources/translations/en.edn

```diff
@@ -141,7 +141,9 @@
   :cart {:add "Add to cart"
          :apply "Apply"
          :apply-for-bundle "Apply for these %1 resources as a bundle "
+         :empty [:div.intro [:p "Select resources to apply for."]]
```

Comment:
You might want to consider using a colon at the end of this text, as well as :continue-existing-application-intro, to make them more inviting. For comparison, see the following images:

![image](https://user-images.githubusercontent.com/49641103/59456310-b9454b00-8e1e-11e9-8545-e4e2d380c95a.png)


![image](https://user-images.githubusercontent.com/49641103/59456290-b0ed1000-8e1e-11e9-8269-d016dc92da22.png)